core/src/main/java/de/erethon/commons/misc/BlockUtil.java

```diff
@@ -166,4 +171,111 @@ public static Block getAttachedBlock(Block block) {
         return blocks;
     }
 
+    /**
+     * @param block the block to check
+     * @return true if the block is an Sign or WallSign, false otherwise
+     */
+    public static boolean isSignOrWallSign(Block block) {
+        return isSign(block) | isWallSign(block);
+    }
+
+    /**
+     * @param block the block to check
+     * @return true if the block is an Sign, false otherwise
+     */
+    public static boolean isSign(Block block) {
```

Comment:
Wenn ich nichts übersehe, sollte isSign() bei 1.13 auch für wall signs true sein:

"Represents a captured state of either a SignPost or a WallSign." - https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/block/Sign.html

Außerdem sollte hier keine Fallunterscheidung nötig sein, weil es BlockStates schon praktisch immer gab: https://jd.andross.fr/1.8.8/org/bukkit/block/Sign.html (für andere alte JavaDocs https://jd.andross.fr/)

Fallunterscheidungen sind nur für das "neue" (1.13+) BlockData-API nötig. Dazu musste ich teilweise auch in die Bukkit1_13-Klasse auslagern. Ich bin mir nicht zu 100% sicher, wann genau.

Reply:
Sollte jetzt passen